packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.integration.test.tsx

```diff
@@ -61,9 +67,10 @@ describe('AccessRequirementList tests', () => {
     // Must be logged in
     await screen.findByText('signed in', { exact: false })
     // Must be certified (since one or more of the ARs requires it)
-    await screen.findByText('certified', { exact: false })
+    await screen.findByText(/You must first become a/)
+    await screen.findAllByText(/certified user/) // may appear multiple times since we also display if the current user is a certified user
     // Must have a verified profile (since one or more of the ARs requires it)
-    await screen.findByText('validated', { exact: false })
+    await screen.findByText('user profile validated', { exact: false })
```

Comment:
Changes in @testing-library/react affected timings so more events are processed by the time we run these queries. In some cases, these matchers are now finding multiple nodes, or nodes that change by the time we query them. Update as needed